server/apps/immich/src/controllers/people.controller.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  AssetResponseDto,
+  AuthUserDto,
+  FacialRecognitionService,
+  ImmichReadStream,
+  mapPerson,
+  PersonResponseDto,
+} from '@app/domain';
+import { Controller, Get, Header, Param, StreamableFile } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { GetAuthUser } from '../decorators/auth-user.decorator';
+
+import { Authenticated } from '../decorators/authenticated.decorator';
+import { UseValidation } from '../decorators/use-validation.decorator';
+import { UUIDParamDto } from './dto/uuid-param.dto';
+
+function asStreamableFile({ stream, type, length }: ImmichReadStream) {
+  return new StreamableFile(stream, { type, length });
+}
+
+@ApiTags('People')
+@Controller('people')
+@Authenticated()
+@UseValidation()
+export class PeopleController {
+  constructor(private service: FacialRecognitionService) {}
+  @Get()
+  async getAllPeople(@GetAuthUser() authUser: AuthUserDto): Promise<PersonResponseDto[]> {
```

Comment:
The `async` keywords con be committed everywhere in this file, since we don't use `await`.